modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -269,10 +271,36 @@ class NDB_Form_User_Accounts extends NDB_Form
             // END multi-site UPDATE
 
             // EXAMINER UPDATE
+            // get all fields that are related to examiners
+            $ex_curr_sites  = array();
+            $ex_prev_sites  = array();
+            $ex_radiologist = 'N';
+            $ex_pending     = 'Y';
+
+            foreach ($values as $k => $v) {
+                //examiner fields
+                if (preg_match("/^ex_[0-9]+$/", $k)) {
+                    //get centerID
+                    $parse_key = explode('_', $k);
+                    $cid       = $parse_key[1];
+                    $ex_curr_sites[$k] = $cid;
+                }
+                if ($k === 'examiner_radiologist') {
+                    $ex_radiologist = $v;
+                }
+                if ($k === 'examiner_pending') {
+                    $ex_pending = $v;
+                }
+            }
+
```

Comment:
section brought out from inside the if statement. This allows to verify if any examiner manipulation is even required. if no checkboxes are checked than there is no reason to go in